templates/blockonomics_checkout.php

```diff
@@ -36,69 +39,90 @@
         
         <!-- Payment Expired -->
         <div class="bnomics-order-expired-wrapper">
-            <h3 class="warning bnomics-status-warning"><?=__('Payment Expired', 'blockonomics-bitcoin-payments')?></h3><br/>
+            <h3><?=__('Payment Expired', 'blockonomics-bitcoin-payments')?></h3><br/>
             <p><a href="#" id="bnomics-try-again"><?=__('Click here to try again', 'blockonomics-bitcoin-payments')?></a></p>
         </div>
 
-        <!-- Blockonomics Checkout Panel -->
+        <!-- Blockonomics Checkout Panel -->    
         <div class="bnomics-order-panel">
-            <div class="bnomics-order-info">
-                <div class="bnomics-bitcoin-pane">
-                    <div class="bnomics-btc-info">
-                        <!-- Left Side -->
+            <table>
+                <tr>
+                    <th class="bnomics-header">
+                        <!-- Order Header -->
+                        <strong><?=__(strtolower($crypto['name']), 'blockonomics-bitcoin-payments')?></strong>
+                        
+                        <div>
+                            <span class="blockonomics-icon-cart"></span>
+                            <?php echo $order['value'] ?> <?php echo $order['currency'] ?>
+                        </div>
+                    </th>
+                </tr>
+            </table>
+            <table>
+                <tr>
+                    <th>
+                        <!-- Order Address -->
+                        <strong class="bnomics-address-text">To Pay, send <?=__(strtolower($crypto['name']), 'blockonomics-bitcoin-payments')?> at this address:</strong>
+                        <strong class="bnomics-copy-address-text"><?=__('Copied to clipboard', 'blockonomics-bitcoin-payments')?></strong>
+                        <div class="bnomics-copy-container">
+                            <input type="text" value="<?php echo $order['address']; ?>" id="bnomics-address-input" readonly/>
+                            <a href="#" id="bnomics-address-copy" class="blockonomics-icon">
```

Comment:
Surrounding the icons in links is not required and can be removed as this adds unexpected behaviour with some themes due to additional css such as :focus, :clicked etc.
For example, the focus styling remains after the qr code has been clicked.
![Focus Issue](https://user-images.githubusercontent.com/39666372/195368511-0257c608-b386-4fc8-8a74-7c2216ec01aa.png)
The same applies to the Refresh Now `<button>`, as wrapping the button with a link should not be required.